### PR TITLE
Style rules corrected, trailing spaces removed, tabs replaces by spaces

### PR DESCRIPTION
**Pull request by it-praktyk**
The backport of #691 
The fix of #689

### CODE REVIEW

File: Pester.Tests.ps1 [hunk 1]
```diff
@@ -146,6 +146,28 @@ Describe 'Style rules' {
         }
     }
 
+    It 'Pester source files lines start with a tab character' {
```
Comment (nohwnd) [inline Pester.Tests.ps1]: Shouldn't this be "Pester source file lines do not start with tab character" or something along the lines of: "Indentation is done with spaces, not tabs."

Comment (it-praktyk) [inline Pester.Tests.ps1]: I agree.

The test was copied directly from the master branch.

**Comment by it-praktyk** [comment]
For to be sure. 

Should I update my PR or  will you apply a proposed patch?

Thx.

**Comment by nohwnd** [comment]
I'll fix it.

**Pull request merged**